### PR TITLE
Remove global angular `irida`

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
The most recent version of IRIDA does not use angularjs on every page.  This merge request stops loading it to every page.

## Related issue
Related to #477 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/pages/template/page.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <head>
   <link rel="stylesheet" th:href="@{/dist/css/app.bundle.css}">
   <style>
-    .ng-cloak {
+    [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak {
```
[the remark this answers is not in the document]
Comment (tom114) [reply src/main/webapp/pages/template/page.html:8]: Seeing we've already had the issue with one item (the sample edit button) we need to check that it didn't magically happen on other pages.  Might have to grep through the project to ensure that there isn't anythign else effected here.

File: src/main/webapp/pages/projects/export/ncbi.html [hunk 2]
```diff
@@ -414,6 +410,8 @@ <h4 class="modal-title" id="Help" th:text="#{project.export.library_selection.ti
           contentType: 'application/json',
           data: JSON.stringify(submission),
           success: function(data, textStatus, jqXHR) {
+            console.log("HERE");
+            console.log(data);
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/pages/projects/export/ncbi.html:414]: Removed in ad430a1ffce884c415935cc87e348609fedca1f9

File: src/main/webapp/pages/projects/export/ncbi.html [hunk 3]
```diff
@@ -41,13 +41,8 @@ <h2 th:text="${project.label}">_project id_</h2>
               <p class="help-block" th:text="#{project.export.namespace.description}">_Namespace used for identifier creation_</p>
             </div>
             <div class="form-group col-md-6">
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/pages/projects/export/ncbi.html:43]: Updated name in cfb5828c1d8f018d171fbc61d5cfe836293d3a3d

File: src/main/webapp/pages/template/base.html [hunk 4]
```diff
@@ -11,12 +11,13 @@
...
-<body vocab="http://schema.org/" typeof="WebPage" ng-app="irida">
+<body>
 <th:block layout:fragment="content">
 </th:block>
 
 <th:block th:if="${analytics}">
 	<th:block th:utext="${analytics}"></th:block>
 </th:block>
 </body>
+</body>ng
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/pages/template/base.html:22]: Oops, sorry.  Removed in 8e302650c1f158500c28c21eb148c517ecbf76ab

File: src/main/webapp/pages/template/page.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/pages/template/page.html:8]: Updated (e9bfa90fe5bde919cbb8df396551369c74e853e6) to the simple `.ng-cloak` class that it was before and added a comment.  Should really not cause any problems as is.  That button was strange, must have been controlled by angular once upon a time.

File: src/main/webapp/resources/js/pages/activities/activities.js [hunk 5]
```diff
@@ -4,4 +4,4 @@
 import angular from "angular";
 import { EventsModule } from "../../modules/events/events";
 
-angular.module("irida").requires.push(EventsModule);
+angular.module("irida.eves", [EventsModule]);
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/activities/activities.js:7]: Fixed in 4511d31b1f10cbad6c08b9c4e08c4df38630bf82, good catch as it was not set on the page either (also updates)

[the remark this answers is not in the document]
Comment (tom114) [reply src/main/webapp/resources/js/pages/activities/activities.js]: Something not loading right.  Admin page is blank now, not showing any events.  I don't see anything in the browser console either.

File: src/main/webapp/pages/template/page.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (tom114) [reply src/main/webapp/pages/template/page.html]: Thanks, just curious though, why all the other variants before?

[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/pages/template/page.html]: It's just the default they show on the angular site.

File: src/main/webapp/resources/js/pages/activities/activities.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/activities/activities.js]: Fixed in b3de8ea

**Pull request merged**